modules/core/src/test/java/org/apache/ignite/internal/util/GridMutableLongTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.util;
+
+import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
+import org.junit.Test;
+
+/**
+ * Class for testing {@link GridMutableLong}.
+ */
+public class GridMutableLongTest extends GridCommonAbstractTest {
+    /**
+     * Checking the correctness of object initialization.
+     */
+    @Test
+    public void testInitialization() {
+        assertEquals(0, new GridMutableLong().get());
+        assertEquals(5, new GridMutableLong(5).get());
```

Comment:
To be honest, I don't see the difference between 5 and 10. It seems to me, one assertion can be skipped.

Reply:
I think that an additional check will not hurt.